bin/encore.js

```diff
@@ -12,7 +12,7 @@
 
 const parseRuntime = require('../lib/config/parse-runtime');
 const context = require('../lib/context');
-const chalk = require('chalk');
+const chalk = require('chalk').default;
```

Comment:
`.default`?

Reply:
Yeah... it doesn't change anything at runtime since `require('chalk')` is the same thing than `require('chalk').default` but it seems that due to how chalk is typed, VSCode (which uses TypeScript to resolve things) doesn't find its properties if you don't specify the `.default`:

![image](https://user-images.githubusercontent.com/850046/55894399-533c0d80-5bba-11e9-97dc-57252688d7c8.png)

![image](https://user-images.githubusercontent.com/850046/55894408-5800c180-5bba-11e9-9c05-d7c53c527f54.png)